roles/common/tasks/serverspec.yml

```diff
@@ -1,8 +1,12 @@
 ---
+- name: clean all serverspec definition
```

Comment:
We cannot have this task here as it will remove serverspecs if one would run tag ursula run post upgrade. Given this is one time activity (i.e wipe all serverspec) needed part of upgrade we should move this to upgrade.yml play. Add pre_tasks: to https://github.com/blueboxgroup/ursula/blob/master/upgrade.yml#L39 . See below for exmaple - https://github.com/blueboxgroup/ursula/blob/master/upgrade.yml#L113